.golangci.yml

```diff
@@ -0,0 +1,26 @@
+output:
+  sort-results: true
+
+run:
+  build-tags:
+  - withebpf
+  - tools
+  - docs
+  skip-files:
+  # Skip autogenerated files.
+  - ^.*.pb.go$
```

Comment:
According to the documentation, it is not necessary to skip all the autogenerated files because they ["_confidently recognize autogenerated files_"](https://github.com/golangci/golangci-lint/blob/29eedbf6e56508cf1ca52ad0eeed067b1c3a3066/.golangci.example.yml#L36-L44). 

Did you have the change of test such feature? I would like to know if it recognizes `iter*v4_bpfel.go` too.

Reply:
You're right, this is not needed, it seem autodetection does the job :+1: Removing.